pkg/volume/azure_dd/azure_common.go

```diff
@@ -40,7 +40,7 @@ import (
 const (
 	defaultStorageAccountType       = storage.StandardLRS
 	defaultAzureDiskKind            = v1.AzureSharedBlobDisk
-	defaultAzureDataDiskCachingMode = v1.AzureDataDiskCachingNone
+	defaultAzureDataDiskCachingMode = v1.AzureDataDiskCachingReadOnly
```

Comment:
Link to the origina PR that resulted in this change
https://github.com/kubernetes/kubernetes/pull/72229

I need to investigate the implications this new default value.